releases.go

```diff
@@ -306,6 +306,9 @@ func newServiceProcess(release *Release, p *Process) *scheduler.Process {
 	if len(ports) > 0 {
 		env["PORT"] = fmt.Sprintf("%d", *ports[0].Container)
 
+		// gliderlabs registrator
+		labels[fmt.Sprintf("SERVICE_%d_NAME", *ports[0].Container)] = release.App.Name
```

Comment:
Makes sense to me, I think starting with a label here is the best option! If you're happy with it, I'll merge.